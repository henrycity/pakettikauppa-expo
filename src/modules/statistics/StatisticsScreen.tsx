import * as React from 'react'

import Styles from '../../common/Styles'
import { TitleText, View, Text } from '../../common/Themed'
import BottomTabWrapper from '../../common/components/BottomTabWrapper'
import useDeviceType from '../../common/hooks/useDeviceType'

export default function StatisticsScreen(): JSX.Element {
  const { isMobile } = useDeviceType()

  return (
    <BottomTabWrapper>
      <View style={Styles.container}>
        <TitleText>Statistics Screen!</TitleText>
        <View
          style={Styles.separator}
          lightColor="#eee"
          darkColor="rgba(255,255,255,0.1)"
        />
        <Text>You are using {isMobile ? 'mobile' : 'desktop'}.</Text>
      </View>
    </BottomTabWrapper>
  )
}
