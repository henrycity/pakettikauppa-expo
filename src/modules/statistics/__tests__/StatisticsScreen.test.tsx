import React from 'react'
import renderer from 'react-test-renderer'

import StatisticsScreen from '../StatisticsScreen'

it('LoginScreen renders correctly', () => {
  const tree = renderer.create(<StatisticsScreen />).toJSON()

  expect(tree).toMatchSnapshot()
})
