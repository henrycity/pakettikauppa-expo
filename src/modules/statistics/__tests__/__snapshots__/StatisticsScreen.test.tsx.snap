// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      Object {
        "flex": 1,
        "justifyContent": "space-between",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
        },
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        },
      ]
    }
  >
    <Text
      style={
        Array [
          undefined,
          Object {
            "fontFamily": "Rubik-Bold",
            "fontSize": 20,
          },
          Object {
            "color": "#233385",
          },
        ]
      }
    >
      Statistics Screen!
    </Text>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#eee",
          },
          Object {
            "height": 1,
            "marginVertical": 30,
            "width": "80%",
          },
        ]
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "#233385",
          },
          Object {
            "fontFamily": "Muli",
          },
          undefined,
        ]
      }
    >
      You are using 
      desktop
      .
    </Text>
  </View>
</View>
`;
